docs/book/src/user/quick-start.md

```diff
@@ -897,7 +918,7 @@ For more information about prerequisites, credentials management, or permissions
 
 For the purpose of this tutorial, we'll name our cluster capi-quickstart.
 
-{{#tabs name:"tab-clusterctl-config-cluster" tabs:"Azure|AWS|CloudStack|DigitalOcean|Equinix Metal|GCP|Metal3|Nutanix|Kubevirt|OpenStack|vSphere,Docker"}}
+{{#tabs name:"tab-clusterctl-config-cluster" tabs:"Azure|AWS|CloudStack|DigitalOcean|Equinix Metal|GCP|Metal3|Nutanix|Kubevirt|OpenStack|vSphere,vcluster,Docker"}}
```

Comment:
Do you mind to change this into
```suggestion
{{#tabs name:"tab-clusterctl-config-cluster" tabs:"Docker,vcluster,others..."}}
```
And fix tabs accordingly?
A tab so long looks really weird in the preview 
(otherwise I will follow up with another PR)

Reply:
No problem at all! The diff looks a little strange since I needed to reorder the tabs to get the correct content under the correct name.